pkgs/swift2objc/.gitignore

```diff
@@ -43,3 +43,6 @@ coverage/
 
 # Files generated by various OSs.
 .DS_Store
+
+# temp directories used during developement to run & test implementation
+_temp/
```

Comment:
nit: missing newline. I recommend configuring your editor to always add newlines on save.

Reply:
done.